platform-tests/example-apps/healthcheck/elasticsearch.go

```diff
@@ -26,8 +26,7 @@ func elasticsearchHandler(w http.ResponseWriter, r *http.Request) {
 
 func testElasticsearchConnection(tls bool) error {
 	var credentials struct {
-		URI          string `json:"uri"`
-		CACertBase64 string `json:"ca_certificate_base64"`
+		URI string `json:"service_uri"`
```

Comment:
Is there a reason this is different from all the other brokers? I think it would be good for this to be consistent and use the `uri` key in the credentials JSON. This is also what the documentation describes here: https://docs.cloudfoundry.org/services/binding-credentials.html